src/nrepl/repl-start.ts

```diff
@@ -84,13 +85,9 @@ export async function downloadDrams(
 ) {
   await Promise.all(
```

Comment:
Note to self: Consider returning the promise instead of awaiting it. Similar to how we shouldn't have catched the error here, I think it makes sense to push the promises out and up as close to the user as possible.